Probesys/bileto
# tec: Setup translations (English and French)

Changes proposed in this pull request:

- Add symfony/translation package
- Add PHP intl extension
- Provide a make command to extract translations
- Translate existing strings
- Translate strings from `validators` and `security` domains

How to test the feature manually:

1. run the app and check strings are in English
2. change `default_locale` to `fr_FR` in `config/packages/translation.yaml`
3. check strings are now in French
4. revert the `default_locale` to `en_GB`

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
